dll/cpl/powercfg/lang/ro-RO.rc

```diff
@@ -1,5 +1,6 @@
 /*
- * Translators: Petru Dimitriu (petrimetri at gmail dot com)
+ * Translators: Andrei Miloiu
```

Comment:
Same: Add your email and move yourself to the bottom.

Reply:
I think I will just update these to our new standard headers before merging.

Reply:
The same intervention may be applicable to [PR №4699](https://github.com/reactos/reactos/pull/4699) and [PR №4700](https://github.com/reactos/reactos/pull/4700) as well.